fvtest/gctest/GCConfigTest.cpp

```diff
@@ -37,6 +37,17 @@
 #define STRINGFY(str) DO_STRINGFY(str)
 #define DO_STRINGFY(str) #str
 
+const char *gcTests[] = {"fvtest/gctest/configuration/sample_GC_config.xml",
```

Comment:
A quick addition to the discussion, if I may: it's generally considered bad practice to do anything complicated before `initGoogleTest()` because it can "obscure" the test environment. This makes it much more difficult for someone who is *not already familiar* with the tests  (e.g. me :slightly_smiling_face:) to figure out what's going on.

As @charliegracie pointed out, removing tests when debugging can be accomplished easily enough using a filter. Although adding new tests does require re-compilation, I would argue that the tests should be added once and then stay there (maybe marked as `DISABLED`, if it doesn't make sense to run them unless the user explicitly asks for them). Some accompanying documentation of what's special about these tests and how they're useful when debugging would also be tremendously helpful (especially for us newbies :sweat_smile:).